pyramid_zipkin/request_helper.py

```diff
@@ -123,6 +124,10 @@ def is_tracing(request):
     elif 'X-B3-Sampled' in request.headers:
         return request.headers.get('X-B3-Sampled') == '1'
     else:
+        override_func = request.registry.settings.get('zipkin.tracing_override')
+        if override_func and override_func(request):
```

Comment:
Yes, percentage will still be evaluated if the `override_func` returns False.

Shouldn't the user ensure that no exceptions will be thrown by the callback? I see that `set_extra_binary_annotations` does no handling of exceptions. I could just wrap the call in a `try/except: pass` block but that feels wrong. I would think that we want the exception to be raised since it's part of the entire pyramid request.